### PR TITLE
string-manipulation-1 | Samuel Cho

### PR DESCRIPTION
**Pull request by SamuelCho1**
JavaScript string manipulation practice with unit tests.

### CODE REVIEW

**Comment by SamuelCho1** [comment]
Screen Shot 2021-01-29 at 2 55 00 PM
Screen Shot 2021-01-29 at 2 55 12 PM
Screen Shot 2021-01-29 at 2 55 23 PM

File: string-manipulation-1/src/get-words.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/* exported getWords */
+function getWords(string) {
+  var String;
```
Comment (codingcodymiller) [inline string-manipulation-1/src/get-words.js:3]: 💀 💀 💀 💀 💀 

Avoid making variables that start with capital letters. `String` has special meaning in JavaScript, and **_you just blew it up_**. How about `words`?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Nice work so far! please see my note

**Comment by SamuelCho1** [comment]
Screen Shot 2021-01-29 at 5 35 00 PM

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**